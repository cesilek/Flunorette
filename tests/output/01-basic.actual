Array
(
    [0] => Flunorette\Row Object
        (
            [name] => Dan
        )

    [1] => Flunorette\Row Object
        (
            [name] => David
        )

)