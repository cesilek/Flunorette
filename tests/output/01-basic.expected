Flunorette\Row Object
(
    [id] => 1
    [created_by] => 0
    [type] => admin
    [name] => Dan
    [city_id] => 1
)